### DISCUSSION TITLE
Declined Student Developer Pack

### DISCUSSION THREAD

**Discussion by Kesler01** [community/community#65404] [GitHub Education]
### Hi everyone,

After I was declined the first time, I was told to develop my profile to be accepted for the pack. After doing that, I am unable to press continue in order to share my location. I have cleared my cache, logged in on other computers, and ensured I am not on a VPN and this glitch still persists. Any thoughts on how I can successfully reapply as I need it in order to run Copilot for my class.

**Comment by arafajs** [comment]
- Try using a different web browser.
- Try using a different internet connection.
- Try clearing your cookies and browser history.
- Try disabling any ad blockers or extensions that you are using.
- Try contacting GitHub Support for assistance.
